policy/modules/services/dbus.if

```diff
@@ -36,6 +36,29 @@ interface(`dbus_exec',`
 	can_exec($1, dbusd_exec_t)
 ')
 
+########################################
+## <summary>
+##	Execute dbus in the specified domain.
+## </summary>
+## <param name="domain">
+## <summary>
+##	Domain allowed to transition.
+## </summary>
+## </param>
+## <param name="dbus_domain">
+## <summary>
+##	Domain to transition to.
+## </summary>
+## </param>
+#
+interface(`dbus_spec_domtrans',`
```

Comment:
I'd like to avoid interfaces like this wherever possible. Also it's unused?

Reply:
Unused. This is leftovers from a previous revision. Removing.